Libraries/Image/RCTImageLoader.h

```diff
@@ -129,6 +129,14 @@ typedef dispatch_block_t RCTImageLoaderCancellationBlock;
  */
 - (RCTImageLoaderCancellationBlock)getImageSizeForURLRequest:(NSURLRequest *)imageURLRequest
                                                        block:(void(^)(NSError *error, CGSize size))completionBlock;
+/**
+ * Determines whether given image URLs are cached locally. The `requests` array is expected
+ * to contain objects convertible to NSURLRequest. The return value maps URLs to strings:
```

Comment:
Does this match the kind of things that can appear in `requests` on Android?

Reply:
Okay, it's good that it's consistent with the existing cross-platform image APIs